### DISCUSSION TITLE
Ignoring Known covariates

### DISCUSSION THREAD

**Discussion by michelidiego** [autogluon/autogluon#3455] [Q&A]
Hi,

I was trying to do a benchmark comparing the fit of the same date using known covariates vs no covariates at all. The thing is I have my train data and I fit the right model using known_covariates_names= ['var_name'] but then when I run the model to fit without using covariates, autogluon automatically assumes 'var_name' as a past covariate.

Is there a way around this? like a way to ignore the variables when I don't want it to be part of the model? or do I necessary need to have 2 different train data sets?

thanks!

**Comment by shchur** [comment]
Hi, there is currently no way to ignore certain columns on the predictor side, so I would recommend to remove the covariates from the dataframe with
```python
train_data = train_data.drop("var_name", axis=1)
```
